### PR TITLE
chore: add docs for telegraf to handle metrics and logs

### PR DESCRIPTION
**Pull request by kiragoo**

### CODE REVIEW

**Comment by Rory-Z** [comment]
Why we need about telegraf docs？

**Comment by kiragoo** [comment]
> Why we need about telegraf docs？ I don't think telegraf should be in “getting start”, If we have to need it, we should create a separate directory, for example "task"

done

File: docs/zh_CN/tasks/install-emqx-prometheus-plugin.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# 安装 `emqx_prometheus` 插件
```
Comment (Rory-Z) [inline docs/zh_CN/tasks/install-emqx-prometheus-plugin.md:1]:
```suggestion
# 部署 EMQX 插件 （以 `emqx_prometheus` 为例）
```

File: docs/zh_CN/tasks/install-emqx-prometheus-plugin.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+>[参考getting-started部署Operator及EMQX 实例](../getting-started/getting-started.md)
```
Comment (Rory-Z) [inline docs/zh_CN/tasks/install-emqx-prometheus-plugin.md:3]:
```suggestion
## 前置条件：
+ 成功部署 EMQX Operator
+ 成功部署 EMQX 自定义资源
>[参考getting-started部署Operator及EMQX 实例](../getting-started/getting-started.md)
```

File: docs/directory.json [hunk 3]
```diff
@@ -94,6 +107,19 @@
         }
       ]
     },
+    {
+      "title": "任务",
+      "children": [
+        {
+          "title": "配置 emqx-prometheus 插件",
```
Comment (Rory-Z) [inline docs/directory.json:114]:
```suggestion
          "title": "部署 EMQX 插件",
```

**Review by Rory-Z — COMMENTED** [review]
Also to be modified is the en_US docs

File: docs/en_US/tasks/install-emqx-prometheus-plugin.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
+# Install EMQX plugin (eg. install `emqx_prometheus` plugin)
+
+## Requirements
+
++ Install EMQX Operator successfully
++ Install EMQX CRD successfully
```
Comment (Rory-Z) [inline docs/en_US/tasks/install-emqx-prometheus-plugin.md:6]:
```suggestion
+ Install EMQX CR successfully
```

**Review by Rory-Z — COMMENTED** [review]
Looks good for me

**Pull request merged**